site/en/guides/border-router/external-commissioning/join.md

```diff
@@ -39,24 +39,17 @@ fdde:ad11:11de:0:ed8c:1681:24c4:3562
 Test the connectivity between the Joiner device in the Thread network and the
 external internet by pinging a public IPv4 address.
 
-For example, the Well-Known NAT64 prefix of `64:ff9b::/96` and an IPv4 address
-of `8.8.8.8` combine to form an IPv6 address of `64:ff9b::808:808`.
-
-Add an external route for the NAT64 Prefix:
-
-```
-$ sudo ot-ctl route add 64:ff9b::/96 s med
-Done
-$ sudo ot-ctl netdata register
+```console
```

Comment:
remove "console" here please, since it's an OT CLI prompt, it should just be the 3 backticks like the other examples